src/app/settings/admin/constraint-templates/constraint-template-dialog/template.html

```diff
@@ -12,30 +12,15 @@
 -->
 <km-dialog-title>{{data.title}}</km-dialog-title>
 <mat-dialog-content>
-  <form [formGroup]="form"
-        fxLayout="column">
-    <mat-form-field fxFlex>
-      <mat-label>Template Name</mat-label>
-      <input required
-             matInput
-             [formControlName]="controls.TemplateName"
-             type="text"
-             autocomplete="off"
-             cdkFocusInitial>
-      <mat-error *ngIf="form.get(controls.TemplateName).hasError('required')">
-        Template Name is <strong>required</strong>.
-      </mat-error>
-    </mat-form-field>
-  </form>
-
   <div class="spec">Spec</div>
   <km-editor [(model)]="spec"
              header="YAML"></km-editor>
+  <mat-hint *ngIf="!isValid()">crd.spec.names.kind needs to be specified</mat-hint>
```

Comment:
I'm not entirely happy with this sentence, but it tells what is missing. Better ideas are welcome.

Reply:
Is it validation for the provided yaml? I thought the reason why we offer YAML editor is to not validate it on our side and make API validate it on submit. Do we really need it?

Reply:
Yeah generally thought the same...
The reason why I thought it would make sense to add this hint was because we generate the name out of this `crd.spec.names.kind` as it has to be the same, only in lowercase...
WDYT?

Reply:
Does it mean that this will be a valid constraint template?

```yaml
crd:
  spec:
    names:
      kind: asd
```

Reply:
it seems like you could add it like this... if it is a _real_ one - I'm not sure, but then validation on backend side isn't entirely working

Reply:
If it isn't really valid then our validation also does not make much sense now. I think API should be updated to validate the initial request and make sure it is valid.